app/src/main/java/com/beemdevelopment/aegis/ui/GroupManagerActivity.java

```diff
@@ -126,12 +169,19 @@ private void saveAndFinish() {
 
             saveAndBackupVault();
         }
+        if (!_renamedGroups.isEmpty()) {
+            for (VaultGroup renamedGroup : _renamedGroups) {
+                _vaultManager.getVault().renameGroup(renamedGroup);
```

Comment:
This causes a crash when you remove the group that you just renamed.

Reply:
Thanks for the info, Will be looking onto it.